### DISCUSSION TITLE
Ask.Copilot not working correctly for C# code windows

### DISCUSSION THREAD

**Discussion by Hefaistos68** [community/community#84128] [Copilot Conversations]
### Select Topic Area

Bug

### Body
Visual Studio 2022, 17.8.3, Copilot 0.1.1986.61475
As you can see in the screen capture below, when invoking Ask.Copilot in a C# code window, its not possible to set the input focus to the copilot window and all keystrokes go into the editor window. This worked before the last version, and it does behave equally in varying VS 2022 editions.

![copilot](https://github.com/community/community/assets/8855055/2cc52e1a-6a7e-466a-bf73-c24b28cc1bfb)

**Comment by ayim** [comment]
@rheapatel23's the expert on most things Copilot Chat in VS, what do ya think?

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
aayim@microsoft.com

**Comment by rheapatel23** [comment]
Hi @Hefaistos68, Thanks for sharing, would you be able to file a bug : https://learn.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2022. Our team will take a look to see try and resolve this. 

Thanks,
Rhea - Product Manager GitHub Copilot for Visual Studio
rhpatel@microsoft.com
